en/research-summaries/conversation-design.md

```diff
@@ -0,0 +1,617 @@
+
+We built a phone number look-up wizard as well as an experimental agent that worked as both a chatbot agent on a screen, and a voice agent, so we could compare the design processes and outcomes for each.
+
+## Definitions
+
+Before we dive into our experiment, let’s go over a few definitions. As this field evolves quickly, definitions tend to change often. 
+
+### Multi-turn interactions 
+
+Multi-turn interactions are those where some kind of back and forth sequence occurs to get to the right answer. They require logic in the sequence to help people only get the answer relevant to their situation.
+
+Wizards are an example of multi-turn interaction.
+
+### Wizards
+
+Wizards use a set of interactive questions to help people work through options to get the answer they need. 
+
+Wizards are becoming fairly common. They have been shown to increase task success in several of our optimization projects.
+
+- [Blog post: The magic of using interactive questions](https://blog.canada.ca/2021/04/08/using-interactive-questions)
+
+### Conversational agents: chatbots and voice agents
+
+For our purposes, we will define conversational agents as virtual agents that use conversation to arrive at answers for people. 
+
+The conversation can be:
+
+- text-based (like in a chatbot on a web page)
+- voice-based (like when using a smart speaker)
+- some form of hybrid (like using Google Assistant on your phone)
+
+Conversational agents take the interaction with users further than a wizard. They allow people to ask questions in their own words. Conversations follow the user’s flow, rather than forcing them into a specific sequence. 
+
+A well-designed voice interaction design seems like a usability dream: 
+
+- people don’t have to learn the design - they already know how to speak
+- people can speak in their own words, rather than being forced to use the words you put on a web page
+- voice interaction is accessible to people with visual impairments
+
+## How conversational agents work
+
+For this project, we used a Google tool called DialogFlow™. The terminology we’re using comes from this tool. Terminology may vary using other tools, but the principles remain the same. 
+
+Conversational agents work like this: 
+
+- the user types or says something to express what they want to know - *“What number should I call with a question about my tax account?”*
+- the agent matches what the user expressed to the set of “intents” pre-designed into the agent 
+  - the set of possible responses is based on the context of the conversation - on what the user has already said, and the answers the agent has already provided
+- the agent replies to the user with a pre-designed response for that situation - *“Do you want to call about your personal tax account, or a business tax account?”*
+- the cycle starts again, if the conversation is not over
+
+
+
+<figure>

```

Comment:
I'm wondering if it's not better to keep the images  in Markdown instead.

Reply:
I am going through the headings now! :)